content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: blog
+title: "The Evolution of Kubernetes Dashboard"
+date: 2021-03-08
+slug: the-evolution-of-kubernetes-dashboard
+---
+
+Authors: Marcin Maciaszczyk, Kubermatic & Sebastian Florek, Kubermatic
+
+In October 2020, the Kubernetes Dashboard officially turned five. As main project maintainers, we barely could believe that so much time has passed since our very first commit to the project. However, looking back with a bit of nostalgia, we had to realize that quite a lot has happened since then. Now it’s due time to celebrate “our baby” with a short recap.
```

Comment:
```suggestion
In October 2020, the Kubernetes Dashboard officially turned five. As main project maintainers, we can barely believe that so much time has passed since our very first commits to the project. However, looking back with a bit of nostalgia, we realize that quite a lot has happened since then. Now it’s due time to celebrate “our baby” with a short recap.
```
minor non-blocking nits :) Just think it helps improve the readability.